src/components/views/styl/Experiment.styl

```diff
@@ -37,11 +37,25 @@ $grid-gap = 30px;
         grid-template-areas:
             "top top"\
             "bottom-left bottom-right";
+
+        #experiment-counts {
+            grid-area: bottom-left;
+        }
     };
 
     &.without-description {
         grid-template-areas:
             "bottom-left bottom-right";
+
+        #experiment-counts {
+            display: grid;
+            grid-template-columns: 1fr 1fr;
+            grid-column-gap: $grid-gap;
```

Comment:
This is acting a little weird. Try turning on the grid display for the `#experiment-details` selector in Developer Tools to see it in more detail. Both the clients and the counts are in the first column, and the second column is just empty space.

![](https://user-images.githubusercontent.com/933396/36576087-77559dcc-181c-11e8-864e-cc4510b307cc.png)

To get the desired effect, we may need to do something like this...

```css
#experiment-counts {
    display: grid;
    grid-template-areas:
        "top top"
        "bottom-left bottom-right";
}
```

... and then assign the `h4` to the top area, the clients to the bottom-left area, and the pings to the bottom-right area.